ci/pipelines/apps-test-ecr.yml

```diff
@@ -443,43 +454,43 @@ jobs:
           image_resource:
             type: registry-image
             source:
-              repository: govukpay/terraform-runner
+              repository: hashicorp/terraform
+              tag: 0.13.4
           params:
-            AWS_ACCESS_KEY_ID: ((readonly_access_key_id))
-            AWS_SECRET_ACCESS_KEY: ((readonly_secret_access_key))
-            AWS_SESSION_TOKEN: ((readonly_session_token))
-            AWS_ASSUME_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
-            AWS_REGISTRY_ID: "((pay_aws_test_account_id))"
+            AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
+            AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
+            AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
             AWS_REGION: eu-west-1
-            AWS_PAGER: ""
-            AWS_DEFAULT_OUTPUT: text
+            TAG: ((.:tag))
           run:
-            path: /bin/bash
+            path: /bin/sh
             args:
               - -ec
               - |
-                assumed_role=($(aws sts assume-role \
-                       --role-arn $AWS_ASSUME_ROLE_ARN \
-                       --role-session-name ecr-test-session \
-                       --query 'Credentials.[AccessKeyId,SecretAccessKey,SessionToken]' \
-                       ) )
-
-                export AWS_ACCESS_KEY_ID=${assumed_role[0]}
-                export AWS_SECRET_ACCESS_KEY=${assumed_role[1]}
-                export AWS_SESSION_TOKEN=${assumed_role[2]}
-
-                application_image_tag=$(cat toolbox-ecr-registry-test/tag)
-
                 cd pay-infra/provisioning/terraform/deployments/test/test-12/microservices_v2/toolbox
                 terraform init
-                terraform plan \
-                  -var application_image_tag=$application_image_tag \
-                  -var nginx_image_tag=''
                 terraform apply \
-                  -var application_image_tag=$application_image_tag \
+                  -var application_image_tag=$TAG \
```

Comment:
shellcheck might ask that this be enclosed in `"` to avoid splitting (although I'm assuming `$TAG` might never have spaces etc

Reply:
yes I've seen this, might as well enclose it in quotes